lib/aasm/instance_base.rb

```diff
@@ -49,6 +49,8 @@ def events(options={})
       state = options[:state] || current_state
       events = @instance.class.aasm(@name).events.select {|e| e.transitions_from_state?(state) }
 
+      events.reject!{|e| Array(options[:reject]).include?(e.name)}
```

Comment:
Could you please change that to

```ruby
options[:reject] = Array(options[:reject])
events.reject! { |e| options[:reject].include?(e.name) }
```

to make it more readable (more spaces) and better performing (making sure it's an array only once)?